app/build.gradle.kts

```diff
@@ -67,14 +67,14 @@ android {
 
 dependencies {
 
-    implementation(platform("androidx.compose:compose-bom:2023.05.01"))
-    implementation("androidx.activity:activity-compose:1.7.1")
+    implementation(platform("androidx.compose:compose-bom:2023.08.00"))
```

Comment:
Could you please check the latest version, if we are changing let us target the latest!
I think it is 10.01